package.json

```diff
@@ -65,14 +65,14 @@
     "del": "^6.0.0",
     "empty": "^0.10.1",
     "env-ci": "2.0.1",
-    "eslint": "^7.18.0",
-    "eslint-config-standard": "^16.0.2",
-    "eslint-import-resolver-babel-module": "^5.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^23.1.1",
+    "eslint": "^8.4.1",
```

Comment:
Good stuff, although I must say it might've been more smart to do these project-level updates within a separate dedicated PR.